regression/goto-instrument/ada-checks/test.desc

```diff
@@ -0,0 +1,9 @@
+CORE
+main.c
+--ada-checks assert-then-assume-first-arg
+^EXIT=10$
+^SIGNAL=0$
+^\[1\] Ada Overflow check: FAILURE$
```

Comment:
The goto-functions line looks like this:
`ASSERT i > 5 // Ada Overflow check`
but somehow the asserted condition is not included in the CBMC output.

Reply:
Fixed.